--- origsrc/sg3_utils-1.31/lib/Makefile.am	2009-06-15 16:46:24.000000000 -0500
+++ src/sg3_utils-1.31/lib/Makefile.am	2011-03-08 03:38:55.967835800 -0600
@@ -138,7 +138,7 @@
 
 lib_LTLIBRARIES = libsgutils2.la
 
-libsgutils2_la_LDFLAGS = -version-info 2:0:0
+libsgutils2_la_LDFLAGS = -version-info 2:0:0 -no-undefined
 
 libsgutils2_la_LIBADD = @GETOPT_O_FILES@ @os_libs@
 libsgutils2_la_DEPENDENCIES = @GETOPT_O_FILES@
