--- origsrc/sg3_utils-1.31/doc/Makefile.am	2010-12-05 20:27:23.000000000 -0600
+++ src/sg3_utils-1.31/doc/Makefile.am	2011-03-08 03:48:29.057717400 -0600
@@ -26,7 +26,7 @@ distclean-local:
 	rm -f sg_scan.8
 
 sg_scan.8 : sg_scan.8.linux
-	cp -p sg_scan.8.linux sg_scan.8
+	cp -p $< $@
 
 endif
 
@@ -50,7 +50,7 @@ distclean-local:
 	rm -f sg_scan.8
 
 sg_scan.8 : sg_scan.8.win32
-	cp -p sg_scan.8.win32 sg_scan.8
+	cp -p $< $@
 
 endif
 
@@ -74,7 +74,7 @@ distclean-local:
 	rm -f sg_scan.8
 
 sg_scan.8 : sg_scan.8.win32
-	cp -p sg_scan.8.win32 sg_scan.8
+	cp -p $< $@
 
 endif
 
--- origsrc/sg3_utils-1.31/lib/Makefile.am	2009-06-15 16:46:24.000000000 -0500
+++ src/sg3_utils-1.31/lib/Makefile.am	2011-03-08 03:38:55.967835800 -0600
@@ -133,7 +133,7 @@ EXTRA_libsgutils2_la_SOURCES = \
 ## CC = clang++
 
 # -std=<s> can be c99, c11, gnu11, etc. Default is gnu89 (gnu90 is the same)
-AM_CFLAGS = -I ../include -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Wall -W
+AM_CFLAGS = -I$(top_srcdir)/include -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Wall -W
 # AM_CFLAGS = -I ../include -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Wall -W -pedantic -std=c11
 
 lib_LTLIBRARIES = libsgutils2.la
--- origsrc/sg3_utils-1.31/src/Makefile.am	2010-12-05 20:27:23.000000000 -0600
+++ src/sg3_utils-1.31/src/Makefile.am	2011-03-08 03:45:46.363565800 -0600
@@ -22,7 +22,7 @@ distclean-local:
 	rm -f sg_scan.c
 
 sg_scan.c : sg_scan.c.linux
-	cp sg_scan.c.linux sg_scan.c
+	cp $< $@
 
 endif
 
@@ -43,7 +43,7 @@ distclean-local:
 	rm -f sg_scan.c
 
 sg_scan.c : sg_scan.c.win32
-	cp sg_scan.c.win32 sg_scan.c
+	cp $< $@
 
 endif
 
@@ -64,7 +64,7 @@ distclean-local:
 	rm -f sg_scan.c
 
 sg_scan.c : sg_scan.c.win32
-	cp sg_scan.c.win32 sg_scan.c
+	cp $< $@
 
 endif
 
@@ -114,7 +114,7 @@ bin_PROGRAMS = \
 ## CC = clang++
 
 # -std=<s> can be c99, c11, gnu11, etc. Default is gnu89 (gnu90 is the same)
-AM_CFLAGS = -I ../include -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Wall -W
+AM_CFLAGS = -I$(top_srcdir)/include -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Wall -W
 # AM_CFLAGS = -I ../include -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Wall -W -pedantic -std=c11
 
 sg_dd_SOURCES =	sg_dd.c
