--- origsrc/sg3_utils-1.34/include/sg_pt_win32.h	2011-08-06 22:10:30.000000000 -0500
+++ src/sg3_utils-1.34/include/sg_pt_win32.h	2012-11-01 21:39:48.563781200 -0500
@@ -10,7 +10,11 @@
  * from msdn.microsoft.com .
  */
 
+#define _WIN32_WINNT 0x0600
 #include <windows.h>
+#ifdef __MINGW64_VERSION_MAJOR
+#include <storduid.h>
+#endif
 
 #define SCSI_MAX_SENSE_LEN 64
 #define SCSI_MAX_CDB_LEN 16
@@ -111,9 +115,11 @@ typedef struct {
 /*
  * file access values
  */
+#ifndef __MINGW64_VERSION_MAJOR
 #define FILE_ANY_ACCESS         0
 #define FILE_READ_ACCESS        0x0001
 #define FILE_WRITE_ACCESS       0x0002
+#endif
 
 
 #define IOCTL_SCSI_BASE    0x00000004
@@ -128,8 +134,10 @@ typedef struct {
 /*
  * Standard IOCTL define
  */
+#ifndef __MINGW64_VERSION_MAJOR
 #define CTL_CODE(DevType, Function, Method, Access)             \
         (((DevType) << 16) | ((Access) << 14) | ((Function) << 2) | (Method))
+#endif
 
 #define IOCTL_SCSI_PASS_THROUGH         CTL_CODE(IOCTL_SCSI_BASE, 0x0401, METHOD_BUFFERED, FILE_READ_ACCESS | FILE_WRITE_ACCESS)
 #define IOCTL_SCSI_MINIPORT             CTL_CODE(IOCTL_SCSI_BASE, 0x0402, METHOD_BUFFERED, FILE_READ_ACCESS | FILE_WRITE_ACCESS)
--- origsrc/sg3_utils-1.34/src/sg_scan.c.win32	2010-03-12 10:48:12.000000000 -0600
+++ src/sg3_utils-1.34/src/sg_scan.c.win32	2012-11-01 21:30:04.655035800 -0500
@@ -37,6 +37,7 @@ static char * version_str = "1.08 (win32
 
 // IOCTL_STORAGE_QUERY_PROPERTY
 
+#ifndef __MINGW64_VERSION_MAJOR
 #define FILE_DEVICE_MASS_STORAGE    0x0000002d
 #define IOCTL_STORAGE_BASE          FILE_DEVICE_MASS_STORAGE
 #define FILE_ANY_ACCESS             0
@@ -115,6 +116,7 @@ typedef struct _STORAGE_PROPERTY_QUERY {
     STORAGE_QUERY_TYPE QueryType;
     UCHAR AdditionalParameters[1];
 } STORAGE_PROPERTY_QUERY, *PSTORAGE_PROPERTY_QUERY;
+#endif
 
 
 /////////////////////////////////////////////////////////////////////////////
